### ISSUE TITLE
[FEATURE]: Use CSS from deliberation-empirica in render window

### ISSUE THREAD

**Issue by JamesPHoughton** [Watts-Lab/researcher-portal#56]
### User Story

As an experiment developer, I want the render window to show exactly what deliberation-empirica will display, including styles, so I can know how my experiment will appear to participants.

### Feature description

The render window should use the windii styling from deliberation-empirica. 

I'm not sure exactly how to do this, because the researcher portal also has its own styling, so we'd need to figure out how to apply an entirely new style to a particular react component. 

### What would this feature enable?

This would make the render window actually a window into what the participant sees when running an experiment, and would make the portal a lot more useful for developers.

### Priority

Convenience for researchers

**Comment by JamesPHoughton** [comment]
@JamesPHoughton @christabusho @evanping schedule a time later in the week to talk about this.

**Comment by JamesPHoughton** [comment]
@christabusho Has been trying to add the styling from windi in deliberation-empirica using a wrapper on the component. Something needs to be set just on the render panel and not the rest of the display. 
Maybe explore using the deliberation-empirica styles throughout the researcher portal.

**Comment by JamesPHoughton** [comment]
- debug one more error, if that works, great, otherwise we'll package this for the fall.

**Comment by JamesPHoughton** [comment]
Some issues identified:
- windi css config in deliberation-empirica defines a bunch of styles and colors that are unique to empirica, that we don't currently have access to in our tailwind config. This handles the colors for the button, for example.
- empirca core provides several different CSS files that windi config imports wholesale, tailwind does not, and oh, by the way we deleted them when we mocked emprica core, so they arent even there.

Some potential solutions
- we may be able to have windi style the wrapped stage component and its children, even if tailwind is rendering the rest. not sure if we need to set up a specific script to have windi evaluate the empirica CSS and deliberation-empirica code to create a CSS file that is imported into the wrapped stage component.
- we could alternately bring these styles to the top level and apply them to the whole editor.
-
